src/xdp-app-info.c

```diff
@@ -606,7 +606,7 @@ xdp_connection_get_pidfd (GDBusConnection  *connection,
                                                 error);
         }
 
-      g_propagate_error (error, local_error);
+      g_propagate_error (error, g_steal_pointer (&local_error));
```

Comment:
Yikes. Really need some tests for the appinfo stuff but that's going to be a real PITA.